lte/gateway/c/core/oai/tasks/mme_app/mme_app_itti_messaging.c

```diff
@@ -143,8 +143,19 @@ status_code_e mme_app_send_s11_release_access_bearers_req(
   release_access_bearers_request_p =
       &message_p->ittiMsg.s11_release_access_bearers_request;
   release_access_bearers_request_p->local_teid = ue_mm_context->mme_teid_s11;
-  pdn_context_t* pdn_connection = ue_mm_context->pdn_contexts[pdn_index];
-  release_access_bearers_request_p->teid = pdn_connection->s_gw_teid_s11_s4;
+  /* Fetch the 1st PDN context to check if the message should be sent
+   * to spgw/sgw_s8 task
+   */
+  pdn_context_t* pdn_connection = ue_mm_context->pdn_contexts[0];
```

Comment:
I don't think this is safe. How do we know that the first session is not deleted while other sessions might be still there?